lib/rules/block-spacing.js

```diff
@@ -15,7 +16,9 @@ var util = require("../ast-utils");
 module.exports = function(context) {
     var always = (context.options[0] !== "never"),
         message = always ? "Requires a space" : "Unexpected space(s)",
-        sourceCode = context.getSourceCode();
+        sourceCode = context.getSourceCode(),
+        tokensAndComments = sourceCode.tokensAndComments,
+        tokenStore = new TokenStore(tokensAndComments);
```

Comment:
There are 13 methods in `TokenStore`. Do you want me to create the equivalent ones for all of them in `SourceCode`?